specification/oracle/Oracle.Database/operations/exadata/dbNodesOperations.tsp

```diff
@@ -0,0 +1,24 @@
+import "@azure-tools/typespec-azure-resource-manager";
+import "../../../models/exadata/dbNodes.tsp";
+
+using Azure.ResourceManager;
+using Autorest;
+using TypeSpec.Versioning;
+
+namespace Oracle.Database;
+
+@armResourceOperations
+interface DbNodes extends OracleReadOnlyResource<DbNode> {
+  // TODO: Swagger generator bug, cannot be extracted to interface
+  @example("./examples/dbNodes_listByParent.json", "List DbNodes by VM Cluster")
+  listByParent is ArmResourceListByParent<DbNode>;
+
+  @doc("VM actions on DbNode of VM Cluster by the provided filter")
+  @example(
+    "./examples/dbNodes_action.json",
+    "VM actions on DbNodes of VM Cluster"
+  )
+  action is ArmResourceActionAsync<DbNode, DbNodeAction, DbNode>;
```

Comment:
I feel like action is a weird operation name?

Reply:
We are trying to be consistent with Oracle Cloud API as much as possible.

https://docs.oracle.com/en-us/iaas/api/#/en/database/20160918/DbNode/DbNodeAction